### PR TITLE
NOISSUE - Add store and search bar for browsed opc-ua server data

### PR DESCRIPTION
**Pull request by manuio**
Signed-off-by: Manuel Imperiale <manuel.imperiale@gmail.com>

### CODE REVIEW

File: src/app/pages/things/opcua/opcua.component.ts [hunk 1]
```diff
@@ -112,9 +119,10 @@ export class OpcuaComponent implements OnInit {
   opcuaNodes = [];
 
   browseServerURI = '';
-  browseNamespace = '';
-  browseIdentifier = '';
+  browseNamespace = '0';
+  browseIdentifier = 'i=84';
```
Comment (drasko) [inline src/app/pages/things/opcua/opcua.component.ts]: Why is this 84 hard-coded and what is this exactly?

Comment (manuio) [reply src/app/pages/things/opcua/opcua.component.ts]: The standard root OPC-UA server node

File: src/app/pages/things/opcua/opcua.component.html [hunk 2]
```diff
@@ -30,47 +21,55 @@
       </div>
     </div>
   </nb-card-header>
+</nb-card>
+
+<nb-card size="medium">
+  <nb-card-header>
+    <nb-list-item>
```
Comment (darkodraskovic) [inline src/app/pages/things/opcua/opcua.component.html]: Don't know if `nb-list-item` is implicitely a row or we need to say it explicitely.

File: src/app/pages/things/opcua/opcua.component.ts [hunk 3]
```diff
@@ -39,6 +40,12 @@ export class OpcuaComponent implements OnInit {
         title: 'Name',
         addable: true,
         filter: false,
+        valuePrepareFunction: (cell) => {
```
Comment (darkodraskovic) [inline src/app/pages/things/opcua/opcua.component.ts]: From the documentation: 
```This function will be invoked with 2 parameters: cell, row```
Maybe add row param, as we do in other cases where we don't use extra parameters.

File: src/app/pages/things/opcua/opcua.component.html (hunk 2 above)
Comment (manuio) [reply src/app/pages/things/opcua/opcua.component.html]: it's implicit

File: src/app/pages/things/opcua/opcua.component.ts (hunk 1 above)
Comment (drasko) [reply src/app/pages/things/opcua/opcua.component.ts]: Please put this in a comment then, because it is obacure

**Pull request merged**